gemrb/core/Audio.cpp

```diff
@@ -118,4 +118,11 @@ float Audio::GetReverb(unsigned int channel) const
 	return channels[channel].getReverb();
 }
 
+Holder<SoundHandle> Audio::Play(const String& resource, unsigned int channel, const Point& p, unsigned int flags, tick_t *length) {
+	auto mbString = MBStringFromString(resource);
+	auto mbResource = StringView{mbString.c_str()};
```

Comment:
why does `StringView(mbString)` not work?

Reply:
Hm, don't remember any more, will check again.

Reply:
I think there is at least one other case here. if this really doesnt work we should fix it, or at least pass the length to avoid `strlen`

Reply:
Did work, not sure any more what I was facing.